cellpack/autopack/loaders/recipe_loader.py

```diff
@@ -204,14 +240,16 @@ def _read(self):
 
         if (
             "format_version" not in recipe_data
-            or recipe_data["format_version"] != self.latest_version
+            or recipe_data["format_version"] != self.current_version
         ):
-            format_version = (
+            input_format_version = (
                 recipe_data["format_version"]
                 if "format_version" in recipe_data
                 else "1.0"
```

Comment:
we should change this to be "1.0.0" to be consistent with our current version being "2.0.0". Or vice versa, i'm not sure we really need all 3 levels. but it should be consistent either way